cbits/ctermid.c

```diff
@@ -0,0 +1,21 @@
+/* Provide ctermid() if it is missing (as on some Android versions). */
+
+#include "HsUnixConfig.h"
+#ifndef HAVE_CTERMID
```

Comment:
Feel free to make that change at some point, including the occurrences in this file.